remotes.go

```diff
@@ -177,13 +174,23 @@ func (i *remotesRowIter) Close() error {
 }
 
 func remoteToRow(repoID string, config *config.RemoteConfig, pos int) sql.Row {
+	url := ""
```

Comment:
If no URL, it should be `nil`, right?

Reply:
hmm, URLs are strings

Reply:
yep, I meant at SQL table level. If there is no value, we should return NULL instead of empty string.

Reply:
OK, done.